### DISCUSSION TITLE
Release 1.1

### DISCUSSION THREAD

**Discussion by Macronair** [Maerciez/TheMillionaireGame#50] [General]
_**The first 1.1 release with a lot of major/minor changes, some new features, fixes and improvements.**_

_**Caution: This version uses a new database design. This mean that when you used an old version and start version 1.1 with a database created in an older version, the program will need to create the new tables in order to use this version without major issues. The old database tables with all the questions will NOT be deleted.**_
_**With the brand new import tool in the Questions Editor, you can easily transfer each question to the new design (but you need to set the difficulty for every question individual).**_

### General
- Added: Options menu for custom textures, sounds and lifeline preferences. #10 #12 
- Added: Options for automatic showing/hiding of question and money strap.
- Added: Option to always show correct answer on host screen. (Like in the first series of the UK in 1998. Maybe you want to use it.)
- Added: Custom sound cues. #12 
- Added: (Lifelines) Double Dip & Ask The Host #29 #30 #10 
- Added: Configuration file (config.xml) to save some settings from the Options window.
- Added: Final Answer sounds for question 1 to 5. (These are optional)
- Changed: Layout of the Control Panel.
- Fixed: Bug in Double Dip lifeline where music continues playing when first answer is correct. #21 
- Fixed: Final Answer sounds are too short / not looping. #28 

### Host / Guest / TV Screen
- Added: 1920x1080 Resolution support (+ full screen). You can find this in the options menu. #23 
- Changed: Lifeline textures with new refreshed ones. Reason is the new lifelines that are available.
- Changed: Messages for used lifelines in the current question are now replaced to the Host Messages box.

### Questions Editor
- Changed: Questions can now be assigned to a specific level/money amount.
- Added: Questions import tool (from old database tables)
- Fixed: UTF-8 characters support in database. #43 #42 #33 #31 #26 

### Fastest Finger First
- Added: Option to change used port number for the FFF Server.
- Added: Documentation for using the FFF Server and Client programs #40 
- Changed: Three Notes and Thinking sounds are not playing at the same time anymore. #34 
- Changed: Points are replaced with time in seconds/miliseconds for the fastest player. #44 

## Additional Notes
In a later 1.1 update, Ask The Audience will be more advanced with the options to add your own percentages.
And it will check if the 50:50 lifeline is used on the same question to modify its results.

More info about this will come later.

## Join the discussion
Also new: For every major version, there will be a new discussion thread to leave your feedback, questions, tips, answers etc. to help each other setting up their Millionaire instance.
The new discussion thread for version 1.1 can be found here: <insert link>

## Share your content!
And if you are using the software in your stream or video, then feel free to let me know as i'm interested to see how other people are using it. ;)

<hr /><em>This discussion was created from the release <a href='https://github.com/Macronair/TheMillionaireGame/releases/tag/'>Release 1.1.671</a>.</em>

**Comment by ZacharyHerbertWallace** [comment]
Suggestions:
the ability to edit the money tree amounts
the ability to edit amount of questions in the money tree
the ability to edit the amount of safety nets and where they place

**Comment by matdoslb** [comment]
Ah finally the new release !!!
Will try to find some bugs and will let you know ;))

**Comment by Macronair** [comment]
Thanks guys for the feedback and suggestions for new features. 
New updates will come in late December because of treatment by the dental surgeon this week. So I will take a little rest for now and will be back ASAP. ;D

**Comment by ImmanuelJeswyn** [comment]
I am glad you once again showing interest in this project. Just found this update through mail from github. looking forward to check.

**Comment by Macronair** [comment]
Yesterday I uploaded a patched version of the game which fixes the Plus One icon that stays on screen after using it.
Also some performance improvements are made when more than one lifeline is used.

Available for download in the Releases page.
